internal/core/domain/round.go

```diff
@@ -223,7 +228,8 @@ func (r *Round) Sweep(
 		PreconfirmedVtxos: preconfirmedVtxos,
 		Txid:              txid,
 		Tx:                tx,
-		FullySwept:        fullySwept,
+		FullySwept:        r.willBeFullySwept(liquiditySwept),
+		LiquiditySwept:    liquiditySwept,
```

Comment:
shall we return the swept amount instead of the percentage? it seems like we're adding complexity for nothing?